app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const start = DateTime.fromISO(registrationStart);
+  const [timeLeft, setTimeLeft] = useState(start.diff(DateTime.local(), ['days', 'hours', 'minutes', 'seconds']).toObject());
+
+  useEffect(() => {
+    if (timeLeft.days === 0
+      && timeLeft.hours === 0
+      && timeLeft.minutes === 0
+      && timeLeft.seconds === 0) {
+      onTimerEnd();
+    }
+  }, [onTimerEnd, timeLeft.days, timeLeft.hours, timeLeft.minutes, timeLeft.seconds]);
+
+  useEffect(() => {
+    const updateTimer = () => {
+      const now = DateTime.local();
+
+      const diff = start.diff(now, ['days', 'hours', 'minutes', 'seconds']).toObject();
```

Comment:
Use a shared function, see above

Reply:
And on that note, you could apply `Math.floor` inside that util function already, so that you can use the return value of the function as param to `setTimeLeft` directly below

Reply:
done